MapboxNavigation/NavigationMapView.swift

```diff
@@ -329,19 +330,11 @@ open class NavigationMapView: MGLMapView, UIGestureRecognizerDelegate {
             // Animate course view updates in overview mode
             UIView.animate(withDuration: duration, delay: 0, options: [.curveLinear], animations: { [weak self] in
                 guard let point = self?.convert(location.coordinate, toPointTo: self) else { return }
-                self?.userCourseView?.center = point
+                self?.userCourseView.center = point
             })
         }
         
-        if let userCourseView = userCourseView as? UserCourseView {
-            if let customTransformation = userCourseView.update?(location: location, pitch: self.camera.pitch, direction: direction, animated: animated, tracksUserCourse: tracksUserCourse) {
```

Comment:
[Codecov](https://codecov.io/gh/mapbox/mapbox-navigation-ios/compare/4ae1d627619c385a38cbf9903cf334152a0375de...20fb6a0bc84d175c740a5e27e77c05a8b9499d66/changes) is showing a 72-line drop in test coverage in UserCourseView.swift, because `CourseUpdatable.update(location:pitch:direction:animated:tracksUserCourse:)`, `CourseUpdatable.applyDefaultUserPuckTransformation(location:pitch:direction:animated:tracksUserCourse:)`, and `UserPuckStyleKitView.draw(_:)` don’t get called. I wonder if the deletion of this chunk of code has anything to do with it.

Codecov is right about there being a problem. The puck never draws:

<img src="https://user-images.githubusercontent.com/1231218/66965992-87dcc600-f030-11e9-92a8-bf6357069035.png" width="300" alt="puck">